docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,778 @@
+{
```

Comment:
<p>Instead of saying "start index of discord", why not say something like "the discord is the subsequence located at index i". I find "start index" to be hard to grasp</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>How about this:</p><p> </p><p>To find the top discord, one can compute the left matrix profile, PL, and locate the index <code>i</code>  where PL  has the maximum finite value. The discord is the subsequence located at index i, i.e. the subsequence <code>T[i:i+m</code> .</p><p> </p><p> </p><p>?</p>

Reply:
<p>I am good with that</p>